index.js

```diff
@@ -186,6 +188,12 @@ HomeAssistantPlatform.prototype = {
           accessory = new HomeAssistantSwitch(that.log, entity, that, 'input_boolean')
         }else if (entity_type == 'fan'){
           accessory = new HomeAssistantFan(that.log, entity, that)
+        }else if (entity_type == 'sensor'){
+          if (entity.attributes && (entity.attributes.unit_of_measurement == '\u00B0C' || entity.attributes.unit_of_measurement == '\u00B0F') && entity.entity_id != 'sensor.vision_zp3111_multisensor_4in1_dew_point_5'){
+            accessory = new HomeAssistantTemperature(that.log, entity, that);
+          }
+        }else if (entity_type == 'climate'){
+          accessory = new HomeAssistantTemperature(that.log, entity, that);
```

Comment:
Would rather leave the climate stuff out so that it can be implemented as a real thermostat service instead.